mindsdb/integrations/handlers/auto_ts_handler/setup.py

```diff
@@ -0,0 +1,21 @@
+from mindsdb.integrations.libs.const import HANDLER_TYPE
```

Comment:
Contents in this file seem duplicated from `__init__.py`. Probably not the culprit, but would be good to fix this and follow the pattern that other ML handlers show.